src/core/src/package_managers/TdnfPackageManager.py

```diff
@@ -384,11 +386,11 @@ def __init_constants_for_dnf_automatic(self):
 
     def get_current_auto_os_patch_state(self):
         """ Gets the current auto OS update patch state on the machine """
-        self.composite_logger.log("Fetching the current automatic OS patch state on the machine...")
+        self.composite_logger.log("[TDNF] Fetching the current automatic OS patch state on the machine...")
```

Comment:
Technically the source file acronym is [TPM], but nit as its sufficiently equivalent.

Reply:
This will involve changes in this entire class which again goes beyond the scope of this PR. But I can make that one here if you think it's okay